client/src/Tools/_framework/Paths/Activities.jsx

```diff
@@ -354,6 +371,21 @@ export function Activities() {
         content={folder.content}
         allDoenetmlVersions={allDoenetmlVersions}
       />
+      {folderId ? (
+        <Box style={{ marginTop: 15, marginLeft: 15 }}>
+          <Link
+            href={`/activities/${userId}${parentFolderId ? "/" + parentFolderId : ""}`}
+            style={{
+              color: "var(--mainBlue)",
+            }}
+          >
+            {" "}
+            &lt; Back
+          </Link>
+        </Box>
+      ) : (
+        ""
```

Comment:
I recommend using `null` rather than an empty string if you don't want to including anything. An empty string is an actual object, though I'm not sure what React does with it.